4001-5000/LIT4117Interpretation.xml

```diff
@@ -41,9 +44,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <abstract>
-            <p/>
+            <p>A collection of commentaries on Old Testament books (from Genesis to Ruth) in Gǝʿǝz and Amharic with parts showing great variability, 
+            but also sharing a common core (according to R. Cowley).</p>
```

Comment:
Would you mind to encode the title of Cowleys publication in a bibl-ref?

Reply:
I have seen the reference below: bm:Cowley1988Exeget
Do you think, it is redundant in the abstract?

Reply:
I do not think we do it regularly; I personally tend not to overfill the short abstract text with references, as they (not numerous) are below anyway.